src/tests/mod.rs

```diff
@@ -1 +1,14 @@
 mod store;
+
+use once_cell::sync::OnceCell;
+use std::sync::atomic::AtomicU64;
+
+static INSTANCE: OnceCell<AtomicU64> = OnceCell::with_value(AtomicU64::new(0));
+
+// A test helper to get a timestamp which will always increment by 1 when called.
+// This guarantees that the timestamp is always increasing in tests.
+// now_timestamp may return two identical timestamps in consecutive calls.
+pub fn now_timestamp() -> u64 {
```

Comment:
I'm not sure why we need this in testing, as you said `now_timestamp()` may return two identical timestamp when call at the same time?

Reply:
The problem is that the return values of `now_timestamp` may be the same even consecutive calls. This will cause a channel update from one side to override another channel update from another side (internally we used the timestamp with channel outpoint as the storage key to channel updates, so identical timestamp along with identical channel outpoint is an identical storage key). After some discussion with quake, we decided to use even timestamp for node 1 updates, and odd timestamp for node 2 updates. Here is the commit https://github.com/nervosnetwork/fiber/pull/308/commits/2583f557ef3984d3bb6743a8f7c6964dd9d74e79 . So we don't need the monotonicity of `now_timestamp` any more.